code/__DEFINES/obj_flags.dm

```diff
@@ -31,17 +31,19 @@
 // Flags for the clothing_flags var on /obj/item/clothing
 
 #define LAVAPROTECT             (1<<0)
-#define STOPSPRESSUREDAMAGE		(1<<1)	//! SUIT and HEAD items which stop pressure damage. To stop you taking all pressure damage you must have both a suit and head item with this flag.
-#define BLOCK_GAS_SMOKE_EFFECT	(1<<2)	//! blocks the effect that chemical clouds would have on a mob --glasses, mask and helmets ONLY!
-#define MASKINTERNALS		    (1<<3)	//! mask allows internals
-#define NOSLIP                  (1<<4)  //! prevents from slipping on wet floors, in space etc
-#define THICKMATERIAL			(1<<5)	//! prevents syringes, parapens and hypos if the external suit or helmet (if targeting head) has this flag. Example: space suits, biosuit, bombsuits, thick suits that cover your body.
-#define VOICEBOX_TOGGLABLE      (1<<6)  //! The voicebox in this clothing can be toggled.
-#define VOICEBOX_DISABLED       (1<<7)  //! The voicebox is currently turned off.
-#define SHOWEROKAY				(1<<8)	//! prevents you from being stupid if you shower in them
-#define SNUG_FIT                (1<<9)  //! prevents hat throwing from knocking this hat off
-#define EFFECT_HAT              (1<<10) //! For hats with an effect that shouldn't get knocked off ie finfoil
-#define SCAN_REAGENTS           (1<<11) // Allows helmets and glasses to scan reagents.
+#define STOPS_PRESSURE_DAMAGE		(1<<1)  //! SUIT and HEAD items which stop pressure damage. To stop you taking all pressure damage you must have both a suit and head item with this flag.
```

Comment:
This would probably be better off being defined as (`STOPS_LOWPRESSURE_DAMAGE | STOPS_HIGHPRESSURE_DAMAGE`) instead of it's own individual flag